back/spec/models/app_configuration_spec.rb

```diff
@@ -40,4 +40,39 @@
       end
     end
   end
+
+  describe '#sanitize_organization_name' do
+    before do
+      @app_config = described_class.instance
+
+      @app_config.settings = {
+        'core' => {
+          'organization_name' => {
+            'en' => 'City of <script>alert("XSS")</script> Springfield',
+            'fr' => 'South Elyse <img src=x onerror=alert(1)>',
+            'nl' => 'Plain <b>text</b> with <i>formatting</i>'
+          }
+        }
+      }
+    end
+
+    it 'removes all HTML tags from organization_name multiloc' do
+      # Trigger the callback manually (it runs before_validation)
+      @app_config.send(:sanitize_organization_name)
+
+      # Get the sanitized organization name
+      sanitized_name = @app_config.settings.dig('core', 'organization_name')
+
+      # Verify all HTML is removed from each language
+      expect(sanitized_name['en']).to eq('City of alert("XSS") Springfield')
+      expect(sanitized_name['fr']).to eq('South Elyse ')
+      expect(sanitized_name['nl']).to eq('Plain text with formatting')
+
+      # Verify no HTML tags remain
+      sanitized_name.each_value do |value|
+        expect(value).not_to include('<')
+        expect(value).not_to include('>')
+      end
```

Comment:
Not really necessary, since you're already testing all the values with equality above.